tests/Command/MakerCommandTest.php

```diff
@@ -26,10 +26,27 @@ public function testExceptionOnMissingDependencies()
 
         $fileManager = $this->createMock(FileManager::class);
 
-        $command = new MakerCommand($maker, $fileManager);
+        $command = new MakerCommand($maker, 'App', $fileManager);
         // needed because it's normally set by the Application
         $command->setName('make:foo');
         $tester = new CommandTester($command);
         $tester->execute(array());
     }
-}
\ No newline at end of file
+
+    /**
+     * @expectedException \LogicException
+     * @expectedExceptionMessage using a namespace other than "Unknown".
+     */
+    public function testExceptionOnUnknownRootNamespace()
+    {
+        $maker = $this->createMock(MakerInterface::class);
+
+        $fileManager = $this->createMock(FileManager::class);
+
+        $command = new MakerCommand($maker, 'Unknown', $fileManager);
+        // needed because it's normally set by the Application
+        $command->setName('make:foo');
+        $tester = new CommandTester($command);
+        $tester->execute(array());
```

Comment:
I like this test. We'll just need to instead look at the output, instead of the exception when we convert it to just a message.